ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/response/ApplicationSendPipeline.kt

```diff
@@ -12,7 +12,9 @@ import io.ktor.util.pipeline.*
  */
 public open class ApplicationSendPipeline(
     override val developmentMode: Boolean = false
-) : Pipeline<Any, ApplicationCall>(Before, Transform, Render, ContentEncoding, TransferEncoding, After, Engine) {
+) : Pipeline<Any, ApplicationCall>(
+    Before, Content, Transform, Render, ContentEncoding, TransferEncoding, After, Engine
```

Comment:
No need to create a new public phase. You can create it locally and use `pipeline.insertPhaseBefore(...)` API inside your hook.

Reply:
done, thanks